### PR TITLE
Unit-test speedup and bug fix in StreamSizeTrigger

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricTestHelper.java [hunk 1]
```diff
@@ -119,10 +119,12 @@ public static synchronized Injector getInjector(CConfiguration conf, @Nullable S
       configuration.set(Constants.CFG_LOCAL_DATA_DIR, TEMP_FOLDER.newFolder("data").getAbsolutePath());
       configuration.set(Constants.AppFabric.REST_PORT, Integer.toString(Networks.getRandomPort()));
       configuration.setBoolean(Constants.Dangerous.UNRECOVERABLE_RESET, true);
+      // Speed up tests
+      configuration.setLong(Constants.Scheduler.EVENT_POLL_DELAY_MILLIS, 100L);
+      configuration.setLong(Constants.AppFabric.STATUS_EVENT_POLL_DELAY_MILLIS, 100L);
+
       injector = Guice.createInjector(Modules.override(new AppFabricTestModule(configuration, sConf)).with(overrides));
-      if (configuration.getBoolean(Constants.Security.ENABLED) &&
-        configuration.getBoolean(Constants.Security.Authorization.ENABLED)) {
```
Comment (anwar6953) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricTestHelper.java:124]: lol, what was here?

**Review by anwar6953 — APPROVED** [review]

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricTestHelper.java (hunk 1 above)
Comment (chtyim) [reply cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricTestHelper.java:124]: ... don't ask.

Comment (anwar6953) [reply cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricTestHelper.java:124]: I don't think we're authorized to know.

**Comment by ecfm** [comment]
LGTM

**Comment by ecfm** [comment]
Better to remove those unnecessary system overrides in notifications accordingly, such as logical start time in system overrides of stream size notification

**Comment by chtyim** [comment]
Build passed https://builds.cask.co/browse/CDAP-RUT1344

**Pull request merged**